application/src/main/java/org/togetherjava/tjbot/commands/free/FreeHelpChannelsRoutine.java

```diff
@@ -0,0 +1,56 @@
+package org.togetherjava.tjbot.commands.free;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.utils.TimeUtil;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.Routine;
+
+import java.time.OffsetDateTime;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Routine that automatically marks inactive help channels as free.
+ * <p>
+ * The routine runs every 10 minutes, and checks for help channels that have been inactive for more
+ * than 2 hours.
```

Comment:
To prevent the doc from going out-of-date with the actual numbers, you can move both numbers into constants and then reference them here using `for more than {@value Foo} hours`. It will susbtitute the vlaue in the text then.

Reply:
Mhm, where did you learn that from? :p